runeq/resources/stream_metadata.py

```diff
@@ -835,6 +835,9 @@ def get_patient_stream_metadata(
     if not patient_id:
         raise ValueError("must provide a patient_id")
 
+    # Checks whether patient is accessible, otherwise raises NotFoundError.
+    get_patient(patient_id=patient_id, client=client)
```

Comment:
Does the GraphQL response from `getStreamList` not include an error message?

Reply:
No it just returns an empty response, otherwise the original code would have worked actually